### DISCUSSION TITLE
Button in select option? Click does not work

### DISCUSSION THREAD

**Discussion by consigliere23** [orchidjs/tom-select#381] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/6314741/168869588-68fe1954-5127-43a4-80ba-6a69d94e9c59.png)
When click on X on "Test profile" it just selects that option.
I added in jQuery:
`
$(document).on("click", ".delete-column-profile", function () {
	console.log("CLICKED"));
});`
Is there any option to overwrite it and in jQuery to catch click event on this X button?

**Comment by oyejorge** [comment, marked as answer]
See "Custom JavaScript " on https://tom-select.js.org/examples/customization/

**Comment by consigliere23** [comment]
Nice
